llvm/test/Object/AArch64/chained-fixups-header.test

```diff
@@ -1,4 +1,4 @@
-REQUIRES: host-byteorder-little-endian
+REQUIRES: target-byteorder-little-endian
```

Comment:
This doesn't make sense: there is no concept of default target endianness for yaml2obj as yaml2obj is target agnostic. The endianness is determined by the yaml input file's contents.

Similarly, llvm-objdump should handle both big and little endian, with the endianness determined by the input file: there's nothing for lit to check here.